### PR TITLE
Add blog post "What's new with Bzlmod?"

### PR DESCRIPTION
**Pull request by Wyverald**

### CODE REVIEW

File: _posts/2023-07-24-whats-new-with-bzlmod.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
+---
+layout: posts
+title: "What's New with Bzlmod?"
```
Comment (meisterT) [inline _posts/2023-07-24-whats-new-with-bzlmod.md:3]: nit: lower case "new"?

File: _posts/2023-07-24-whats-new-with-bzlmod.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+authors:
+ - wyv
+---
+
+Since the official launch of Bzlmod in Bazel 6.0 in December 2022, the team has been hard at work improving Bzlmod in time for enabling it by default in 7.0, slated to be launched by the end of this year. This blog post will give you an idea of what we've done so far, and what changes you can expect in the future.
```
Comment (meisterT) [inline _posts/2023-07-24-whats-new-with-bzlmod.md:8]: "end of this year" --> November?

File: _posts/2023-07-24-whats-new-with-bzlmod.md (hunk 1 above)
Comment (Wyverald) [reply _posts/2023-07-24-whats-new-with-bzlmod.md]: hmm. it appears that we're not consistent w.r.t title case for blog post titles. (Even Wikipedia lists multiple house rules for title case: https://en.wikipedia.org/wiki/Title_case)

In this specific case, I agree that a lowercase "new" somehow looks better. So changed.

File: _posts/2023-07-24-whats-new-with-bzlmod.md (hunk 2 above)
Comment (Wyverald) [reply _posts/2023-07-24-whats-new-with-bzlmod.md:8]: I didn't want to promise this... 5.0 was released in Jan 2022, and 6.0 was released in Dec 2022. So we might slip into Dec 2023 for 7.0.

**Pull request merged**